cardano-cli/src/Cardano/CLI/EraBased/Query/Run.hs

```diff
@@ -1856,15 +1846,26 @@ runQuerySPOStakeDistribution
           )
           (Map.keys addressesAndRewards)
 
-    let toWrite =
+    let json =
           [ ( spo
             , coin
             , Map.lookup spo spoToDelegatee
             )
           | (spo, coin) <- Map.assocs spoStakeDistribution
           ]
 
-    writeOutput mOutFile toWrite
+        output =
+          outputFormat
+            & ( id
```

Comment:
Is this `id` to get alignment from the formatter? I don't think it has any effect on the code

Reply:
Yes, it is purely for alignment purposes.